From: Karl Berry <karl@freefriends.org>
Date: Sat, 6 Jun 2020 15:47:33 -0700
Subject: tests: support -fno-common in vala-mix2 test.

This change fixes https://bugs.gnu.org/41726.

* t/vala-mix2.sh: extern in .h, initialization in .c.
GCC 10 defaults to -fno-common.
---
 t/vala-mix2.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/t/vala-mix2.sh b/t/vala-mix2.sh
index 8e6a214..933a3ff 100644
--- a/t/vala-mix2.sh
+++ b/t/vala-mix2.sh
@@ -44,7 +44,7 @@ int main ()
 END
 
 cat > foo.h <<'END'
-int foo;
+extern int foo;
 int bar (void);
 int baz (void);
 END
@@ -56,7 +56,7 @@ END
 
 cat > baz.c <<'END'
 #include "foo.h"
-extern int foo = 0;
+int foo = 0;
 int baz (void) { return 0; }
 END
 
